include/deal.II/lac/la_parallel_block_vector.templates.h

```diff
@@ -221,14 +253,34 @@ namespace LinearAlgebra
       AssertDimension(this->n_blocks(), petsc_vec.n_blocks());
       for (unsigned int i = 0; i < this->n_blocks(); ++i)
         {
-          const auto &partitioner  = this->block(i).get_partitioner();
-          IndexSet    combined_set = partitioner->locally_owned_range();
-          combined_set.add_indices(partitioner->ghost_indices());
-          ReadWriteVector<Number> rw_vector(combined_set);
-          rw_vector.import(petsc_vec.block(i), VectorOperation::insert);
```

Comment:
does this mean that using a RWV silently gives the wrong results without triggering any errors?

Reply:
No, it was complaining in the failing test linked.
And for clarification, the code was copied from what we previously had in the `operator=(PETScWrapper::MPI::Vector)` overload for the non-block vector.

Reply:
sorry, I happened to click on the release build which does not show the assert. :+1: